frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts

```diff
@@ -0,0 +1,44 @@
+import "@elements/entry/admin/images/meta/sections/left";
```

Comment:
on the file itself - there is no entry for "imagelabel", should be in `admin/images/meta/`